src/core/include/openvino/op/prior_box.hpp

```diff
@@ -73,7 +73,6 @@ namespace v8 {
 class OPENVINO_API PriorBox : public Op {
 public:
     OPENVINO_OP("PriorBox", "opset8");
-    BWDCMP_RTTI_DECLARATION;
```

Comment:
Hi, @ilyachur : sorry for the late respond(because of vocation). if I didn't misunderstand in previous PR,, I think this BWD macro is suggested to remove(https://github.com/openvinotoolkit/openvino/pull/8114#discussion_r758030855), but if this BWD macro need to keep, I will remove this change from this PR.